### ISSUE TITLE
Confusing Ownership

### ISSUE THREAD

**Issue by ahawkins** [nicolaka/ddc-aws#45]
I came across this repo trying to debug issues with the quick start template provided by AWS. One team member pointed out that the AWS repo (https://github.com/aws-quickstart/quickstart-docker) should be kept in sync with this repo. This is confusing to me because the template is hosted by AWS and their official guides point to their own repo.

Overall I'm confused on the ownership and issue reporting model--especially as a potential contributor :) Can either this project or the AWS project README's be updated? I know this is a collaborative effort between AWS and Docker Inc hence the two repos. Just clarifying things would help users like myself.

Thanks! :bow:

**Comment by nicolaka** [comment]
@ahawkins we're working to address this issue. stay tuned.

**Comment by nicolaka** [comment]
@ahawkins this repo is the right place to open issues and submit PRs. The `aws-quickstart/quickstart-docker` repo is used for running the CI testing for this project. Sorry for any confusion we created but we'll be closing the issues you created in that repo. I opened some of those issues here in this repo.

Not sure if you noticed, but I integrated fixes for issues #5,10,12 already in master which became https://github.com/nicolaka/ddc-aws/releases/tag/v1.1 . We're running some tests right now and plan to release it soon. It should address many of the issues you raised in the repo and blog post.

Thanks again for opening issues and creating PRs! 

cc @bchav @avattathil
